Product/Production/Services/DocumentSubmissionCore/src/test/java/gov/hhs/fha/nhinc/docsubmission/inbound/deferred/request/PassthroughInboundDocSubmissionDeferredRequestTest.java

```diff
@@ -105,27 +89,25 @@ public void convertDataToFileError() throws LargePayloadException {
         AssertionType assertion = new AssertionType();
 
         AdapterDocSubmissionDeferredRequestProxyObjectFactory adapterFactory = mock(AdapterDocSubmissionDeferredRequestProxyObjectFactory.class);
-        XDRAuditLogger auditLogger = mock(XDRAuditLogger.class);
+        DocSubmissionDeferredRequestAuditLogger auditLogger = mock(DocSubmissionDeferredRequestAuditLogger.class);
```

Comment:
Instead of creating objects for each method, for logger and property object, can move it up to a method with @Before annotation to call it before the actual test methods are called?